src/meta/processors/BaseProcessor.inl

```diff
@@ -454,5 +454,21 @@ BaseProcessor<RESP>::indexCheck(const std::vector<nebula::cpp2::IndexItem>& item
     return cpp2::ErrorCode::SUCCEEDED;
 }
 
+template<typename RESP>
+bool BaseProcessor<RESP>::checkIndexExist(const std::vector<std::string>& fields,
+                                          const nebula::cpp2::IndexItem& item) {
+    for (size_t i = 0; i < fields.size(); i++) {
+        if (fields[i] != item.get_fields()[i].get_name()) {
+            break;
+        }
+
+        if (i == fields.size() - 1) {
+            LOG(ERROR) << "Index " << item.get_index_name() << " have existed";
+            return true;
+        }
+    }
+    return false;
+}
+
```

Comment:
Can we handle the error as below ? 
tag (col1, col2 ,col3)
create index on tag (col1, col2, col3)
create index2 on tag (col1, col2, col3, col1)

Reply:
Good Point We should check whether inculcate the duplicate column.